scripts/ci/validateassets.ps1

```diff
@@ -50,54 +50,54 @@ $Directory = Resolve-Path -Path $Directory
 #
 # The folders are relative to the RepoRoot.
 $packages = [ordered]@{
-    "StandardAssets" =  @(
+    "StandardAssets" = @(
         "MRTK/StandardAssets"
     );
-    "Foundation" =      @(
+    "Foundation"     = @(

```

Comment:
Is this (and the other instances of this kind of spacing alignment) typical for powershell files? It's the first I've seen it and it looks kinda janky 😅

Reply:
i guess haha? it autoformatted and aligned with the furthest out name so the `=`s are vertically aligned. I can always revert this though! It's probably more personal preference

Reply:
Looks like we were previously aligning the `@(`s, so this change also shifts the `=`s over